src/main/features/response/index.ts

```diff
@@ -82,13 +64,7 @@ export class Feature {
     app.all(/^\/case\/.+\/response\/(?!confirmation|receipt|summary).*$/,
       DraftMiddleware.requestHandler(new DraftService(), 'response', 100, (value: any): ResponseDraft => {
         return new ResponseDraft().deserialize(value)
-      }),
-      (req: express.Request, res: express.Response, next: express.NextFunction) => {
-        res.locals.draft = res.locals.responseDraft
```

Comment:
I think we still need this bit. other wise a lot of the pages won't have the correct draft I believe?